pxb/jenkins/test-binary-pxb80

```diff
@@ -28,11 +28,10 @@ fi
 
 if [[ -f /usr/bin/apt-get ]]; then
     DIST=$(lsb_release -sc)
-    DEBIAN_FRONTEND=noninteractive sudo -E apt update
```

Comment:
Don't remove `DEBIAN_FRONTEND=noninteractive`, it should be there just in case worker have been re-used and installation of package would involve interactive changes

Reply:
done